tests/kernel/mellanox_ofed.pm

```diff
@@ -31,6 +31,8 @@ sub run {
         zypper_call('--quiet in openvswitch python-devel python2-libxml2-python libopenssl1_1 insserv-compat libstdc++6-devel-gcc7 createrepo_c rpm', timeout => 500);
     }
     elsif (check_var('VERSION', '12-SP4')) {
```

Comment:
I thought we already clarify that we adding SDK during installation, not manually

Reply:
Adding the variable WORKAROUND_MODULES=base,sdk didn't make the magic.. http://loewe.arch.suse.de/tests/1068#step/mellanox_ofed/8

Maybe we are missing some other parameter or some config, I don't know, but I add it back until we know a better solution.